sdk/python/kfp/onprem.py

```diff
@@ -1,41 +1,60 @@
 from typing import Dict
 
 
-def mount_pvc(pvc_name='pipeline-claim', volume_name='pipeline', volume_mount_path='/mnt/pipeline'):
-    """Modifier function to apply to a Container Op to simplify volume, volume mount addition and
-    enable better reuse of volumes, volume claims across container ops.
+def mount_pvc(
+    pvc_name='pipeline-claim',
+    volume_name='pipeline',
+    volume_mount_path='/mnt/pipeline'
+):
+    """Modifier function to apply to a Container Op to simplify volume, volume
```

Comment:
nit: Thanks for adding the docstring. Ideally, there should be a one-line short description, followed by a blank line, then the detailed description.
https://google.github.io/styleguide/pyguide.html#381-docstrings